### ISSUE TITLE
Cmd+Space shortcut

### ISSUE THREAD

**Issue by RumataEstor** [microsoft/vscode#79847]
Issue Type: <b>Bug</b>

I want to map a shortcut "Cmd+Space" to a command in VSCode but "Keyboard Shortcuts" in VSCode doesn't seem to accept it. I removed that shortcut from Spotlight and I am pretty sure nothing in MacOS intercepts it as Emacs on the same system accepts it without a problem.

VS Code version: Code 1.37.1 (f06011ac164ae4dc8e753a3fe7f9549844d15e35, 2019-08-15T16:16:34.800Z)
OS version: Darwin x64 18.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz (8 x 2400)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (1.12GB free)|
|Process Argv|-psn_0_3322667|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-emacs-ergomac|bed|0.8.2
elixir-ls|Jak|0.2.25
git-tree-compare|let|1.7.0
vscode-emacs-friendly|lfs|0.9.0

</details>

**Comment by alexandrudima** [comment]
@RumataEstor What does VS Code detect when you press the key combination? 
![keyboard-input](https://cloud.githubusercontent.com/assets/5047891/25478333/fe31c830-2b40-11e7-865c-b2c3bb10dd75.gif)

**Comment by RumataEstor** [comment]
It did react when "Cmd" was held but when I pressed "Space" it wouldn't do anything.

Anyway, I can't reproduce the problem anymore and now everything works as intended.

I believe it was resolved after VSCode restart and I assume the instance which didn't accept the "Cmd+Space" shortcut was started when the shortcut was globally registered for "Spotlight". Not sure whether it's MacOS issue or VSCode but I know a simple solution for that - just restart VSCode =)

Sorry for the noise.
